### PR TITLE
Implement jv-oop task

### PR DESCRIPTION
**Pull request by vbuchii**

### CODE REVIEW

File: src/main/java/core/basesyntax/machines/Bulldozer.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.machines;
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/machines/Bulldozer.java:1]:
```suggestion
package core.basesyntax.machine;
```
let's better use singular for package names

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]
please don't make PRs from `master` branch
you should push your solution to a different branch and make a PR from it

let's fix this and make a new PR

File: src/main/java/core/basesyntax/machines/Bulldozer.java (hunk 1 above)
Comment (vbuchii) [reply src/main/java/core/basesyntax/machines/Bulldozer.java:1]: There are few different machines. That is why I named that package in the plural. Could you explain please why do you think it's better to use a singular name here?

**Comment by vbuchii** [comment]
@KseniiaMakarova, closing this out.  I've created a new [PR](https://github.com/mate-academy/jv-oop/pull/299) from the feature branch. Please take a look.

**Pull request closed without merging**